cpp/src/io/parquet/reader_impl_helpers.hpp

```diff
@@ -323,6 +355,7 @@ class aggregate_reader_metadata {
   /**
    * @brief Filters the row groups based on predicate filter
    *
+   * @param sources Lists of input datasources.
```

Comment:
Seems like no periods is the style here.
```suggestion
   * @param sources Lists of input datasources
```